### ISSUE TITLE
Uncaught TypeError: Cannot read property 'getAttribute' of undefined

### ISSUE THREAD

**Issue by ricxsar** [Charmatzis/react-leaflet-google#28]
Hi. I'm using the latest version of `react-leaflet-google` and when rendering the tile of the google maps, an error will occur on line `leaflet-src.js:11297`. See below.
```
Uncaught TypeError: Cannot read property 'getAttribute' of undefined
    at NewClass._tileReady (leaflet-src.js:11297)
    at NewClass.<anonymous> (react-leaflet-google.js:1657)
    at NewClass._onMutatedImage (react-leaflet-google.js:1603)
    at NodeList.forEach (<anonymous>)
    at NewClass._onMutations (react-leaflet-google.js:1545)
```

The specific version the libraries dependencies I'm using are:
```
react-leaflet@1.9.1
leaflet@1.3.3
google-maps@^3.3.0
react@^16.4.1
react-leaflet-google@^3.3.1
react-dom@^16.4.1
```
![screenshot from 2018-07-30 14-39-11_2](https://user-images.githubusercontent.com/8968902/43381591-b32e3d06-9407-11e8-858d-df2cbe51e27d.png)

**Comment by ricxsar** [comment]
@Charmatzis Any updates on this? I cannot migrate to react@16 because of this.
I tried to create a very minimal app to check whether it is the problem on my end but it isn't.

**Comment by pajicv** [comment]
I had the same problem. 
It appeared when upgraded to leaflet 1.3.3. 
I solved it by downgrading leaflet to 1.3.1.

**Comment by nighto** [comment]
I'd like to add that downgrading to leaflet 1.3.1 solved it over here.

**Comment by ricxsar** [comment]
It also solve the problem. Now I can migrate to react@16. Thanks for pointing it out @pajicv.

**Comment by RustyRaptor** [comment]
Ok but what's causing this? It's broken in leaflet 2 as well.

**Comment by ricxsar** [comment]
@RustyRaptor I think leaflet@2 is not supported at the moment.
```
supported versions
"react-leaflet": "^1.9.1"
"leaflet": "^1.3.0""
"react": "^15.0.0 || ^16.0.0"
```

I'm closing this issue since the problem was solved by pinning the `leaflet` to version `1.3.1`.

**Comment by RustyRaptor** [comment]
Ah I was thinking react-leaflet for some reason. My bad.
